tests/execution/providers/google.py

```diff
@@ -0,0 +1,68 @@
+"""add -l or --long to the cli args to run this test (disabled by default 
+because too slow)"""
+
+# 3M-TODO test everything
+import numpy as np
+import pytest
+
+from mpqp.core.instruction.measurement import Observable, ExpectationMeasure
+from mpqp.gates import *
+from mpqp import QCircuit
+from mpqp.measures import BasisMeasure
+from mpqp.execution import run
+from mpqp.execution.devices import GOOGLEDevice
+from mpqp.qasm import qasm2_to_cirq_Circuit
+
+import sys
+
+
+@pytest.mark.parametrize(
+    "circuit",
+    [
+        # SAMPLE JOB
+        QCircuit(
+            [
+                T(0),
+                CNOT(0, 1),
+                Ry(np.pi / 2, 2),
+                S(1),
+                CZ(2, 1),
+                SWAP(2, 0),
+                BasisMeasure(list(range(3)), shots=2000),
+            ]
+        ),
+        # STATEVECTOR JOB
```

Comment:
Should we also add an example for Observable job here ? or only keep the one in the observable test ?